src/lib/jsonParseV2/parseEntitiesVector.cpp

```diff
@@ -0,0 +1,217 @@
+/*
+*
+* Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Orion dev team
+*/
+#include <regex.h>
+
+#include <string>
+#include <vector>
+#include <algorithm>
+
+#include "rapidjson/document.h"
+
+#include "alarmMgr/AlarmManager.h"
+#include "alarmMgr/alarmMgr.h"
+#include "rest/ConnectionInfo.h"
+#include "rest/OrionError.h"
+#include "parse/forbiddenChars.h"
+#include "apiTypesV2/EntID.h"
+#include "common/errorMessages.h"
+#include "jsonParseV2/utilsParse.h"
+#include "jsonParseV2/parseEntitiesVector.h"
+
+
+
+/* ****************************************************************************
+*
+* parseEntitiesVector -
+*/
+bool parseEntitiesVector
```

Comment:
This is just a movement of the function from parseSubscription.cpp? Or some additional change were done? I'm explicitely asking as this is difficult to check in github (which shows "all green" or "all red" in the diff and no per-line changes can be identified).

(I'm not sure if the removed line about a unexplicable checking was in this function...)

Reply:
Just a move, except for the removal of the `std::find`, which you are aware of already

Reply:
Ok. NTC